impeller/playground/backend/metal/playground_impl_mtl.mm

```diff
@@ -68,6 +68,15 @@
   }
   data_->metal_layer = [CAMetalLayer layer];
   data_->metal_layer.device = ContextMTL::Cast(*context).GetMTLDevice();
+
+  // Allow shaders to bind the on-screen textures.
+  data_->metal_layer.framebufferOnly = NO;
+  if (@available(macOS 10.13, *)) {
+    // When calling nextDrawable, wait for the texture to become available
+    // indefinitely.
+    data_->metal_layer.allowsNextDrawableTimeout = NO;
```

Comment:
These two layer properties were needed to make the surface textures readily bindable. Both of them are enabled in the iOS Flutter embedder, which enabled Skia to do the same thing. `framebufferOnly` makes the surface textures readable. Without `allowsNextDrawableTimeout`, multi-pass rendering seemed to result in the texture being presented before it finished rendering.

Reply:
We do set framebufferOnly in the Skia renderer but allowsNextDrawableTimeout is definitely not being set. I think we should investigate the need for setting this.

Reply:
Removed this in favor of just rendering to a third texture in this situation.